### DISCUSSION TITLE
Can you no longer install Homebrew on MacOS Catalina 10.15?

### DISCUSSION THREAD

**Discussion by amadsgit** [Homebrew/discussions#5095] [Getting started]
### Output of `brew config`

```shell
terjadi error saar instalasi
```

### Output of `brew doctor`

```shell
error
```

### Description of issue

tidak bisa install homebrew di macos catalina 10.15

**Comment by SMillerDev** [comment]
> Can you no longer install Homebrew on MacOS Catalina 10.15?

You can, but just like with the rest of macOS 10.15 it's not officially supported and if something breaks you're on your own.
